docs/usage/shoot_credentials_rotation.md

```diff
@@ -170,9 +171,50 @@ It is currently **not** rotated automatically and there is no way to trigger it
 
 ## `ServiceAccount` Token Signing Key
 
-This key is used to sign the tokens for `ServiceAccount`s.
-It is currently **not** rotated automatically and there is no way to trigger it manually.
-Note that there is the option to pass such key to Gardener, please see [this document](shoot_serviceaccounts.md#signing-key-secret) for more details.
+Gardener generates a key which is used to sign the tokens for [`ServiceAccount`s](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/).
+Those tokens are typically used by workload `Pod`s running inside the cluster in order to authenticate themselves with the `kube-apiserver`.
+This also includes system components running in the `kube-system` namespace.
+
+The token signing key has no expiration date.
+Since it might require adaptation for the consumers of the `Shoot`, there is no automatic rotation and **it is the responsibility of the end-user to regularly rotate the signing key.**
```

Comment:
Is this signing key the same one that can be configured with the [`kubeAPIServer.serviceAccountConfig.signingKeySecretName.name`](https://github.com/gardener/gardener/blob/cd0fc9f26f6d63d37545b45eb476a2f7858020f2/example/90-shoot.yaml#L184-L185)? If yes, then we could mention that it is the responsibility of the end-user in case they don't configure `kubeAPIServer.serviceAccountConfig.signingKeySecretName.name`. Do you know how the rotation would look like when the Secret is not gardener managed one? We could describe the rotation in that case as well.

Reply:
Currently, no rotation is implemented for user-provided signing keys, and this field gets deprecated with https://github.com/gardener/gardener/pull/5980 and removed eventually. I would not invest anything into it now (also no docs).